### ISSUE TITLE
More Starter Site Options

### ISSUE THREAD

**Issue by johnhenley** [DNNCommunity/Dnn.CommunityForums#725]
The Forums module has a number of features that are not used by default, and some are not obvious:
- Tags
- Categories
- Subforums
- Likes
- Properties 
- Post Icons
- ...

The initial experience creates a quick starter site but could be enhanced by offering choice of different starter sites (e.g., empty, helpdesk, Q&A, etc.) and/or an interview process with configuration questions. Also offer an option to pre-seed some content to better surface features.

**Comment by johnhenley** [comment]
@Timo-Breumelhof @WillStrohl @valadas looking for example use cases from community

**Comment by valadas** [comment]
I don't have any publicly available forums and the use case I have is very basic. 

I do not use Tags, Categories, SubForums, Likes, Properties or Icons myself

**Comment by johnhenley** [comment]
> I don't have any publicly available forums and the use case I have is very basic. 
> 
> 
> 
> I do not use Tags, Categories, SubForums, Likes, Properties or Icons myself

@valadas thanks for replying:)

**Comment by Timo-Breumelhof** [comment]
Tags > yes
Categories > Does that work? if so I would use it
Subforums > Not using these, not going to I think
Likes > yes, more options? :-)
Custom Properties > I tried them, seems to have some bugs, but I so see use cases
Post Icons > Nope

**Comment by Timo-Breumelhof** [comment]
User Mentions would be my first wish :-)

**Comment by Timo-Breumelhof** [comment]
I would see helpdesk and "stackoverflow" (vote answers up) as useful options

**Comment by Timo-Breumelhof** [comment]
@johnhenley it;s a long time ago. IIRC one thing was that you cannot remove a custom property once you created it

**Comment by johnhenley** [comment]
> @johnhenley it;s a long time ago. IIRC one thing was that you cannot remove a custom property once you created it

Hmm. That would make sense since property values are stored as an xml document in the topic table :(. So to remove a property would require unpacking and repacking every topic that has a property being deleted. 

As I've moved through the modernization of the module, moving  properties to separate data structures/tables has been something I've thought about doing. I can see properties being a very useful feature

**Comment by Timo-Breumelhof** [comment]
> > @johnhenley it;s a long time ago. IIRC one thing was that you cannot remove a custom property once you created it
> 
> Hmm. That would make sense since property values are stored as an xml document in the topic table :(. So to remove a property would require unpacking and repacking every topic that has a property being deleted.
> 
> As I've moved through the modernization of the module, moving properties to separate data structures/tables has been something I've thought about doing. I can see properties being a very useful feature

Yes. We are using the Forums as a simple issue tracker for clients. Works Ok. But with a bit enhanced custom properties it could work even better..
